src/plugins/discover/public/application/apps/main/components/sidebar/discover_field_details.tsx

```diff
@@ -111,14 +108,6 @@ export function DiscoverFieldDetails({
           </>
         )}
       </div>
-      {!details.error && showFooter && (
```

Comment:
:information_source: Since we anyway show the footer separate inside the parent component, when multi fields are shown, I removed this dupliate code, and let both cases just handle via the same place.